fcrepo-search-api/src/main/java/org/fcrepo/search/api/SearchResult.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.search.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author dbernstein
+ */
+public class SearchResult {
+    @JsonProperty
+    private PaginationInfo pagination = new PaginationInfo();
+    @JsonProperty
+    private List<Map<String, Object>> results = new ArrayList();
+
+    public SearchResult() {
+
+    }
+
+    public PaginationInfo getPagination() {
```

Comment:
I am surprised that the build did not fail, given the lack of JavaDoc on these `public` methods... something must have regressed in our checkstyle configuration.

Reply:
Me too.  I'm not sure what would have caused the change.